snapfaas/src/fs/mod.rs

```diff
@@ -1031,8 +1031,6 @@ impl<S: BackingStore> FS<S> {
             }
         }
 
-        assert!(visited.iter().map(|o| objects.contains(o)).all(|x| x));
```

Comment:
What's the reason of removing this assertion?

Reply:
It essentially checks the functional correctness of `get_keys()` that it contains all objects reachable from the root. Three things, I think: 1) the assertion isn't directly related to the correctness of GC; 2) If an object key doesn't exist in the backing datastore, an error would return before the assertion because of traversing; 3) GC assumes the FS is initialized (having a root). If not initialized, the assertion fails. A workaround is to check if the root exists, but generally I don't think the assertion is useful.